source/index.js

```diff
@@ -63,6 +63,10 @@ class likely {
      * @param {Object} [options] additional options for each widget
      */
     static initiate(nodes, options) {
+        if (!isBrowserEnv) {
+            return;
+        }
```

Comment:
Why do you use constructor or `initate` method inside React component? I mean, with this check it does nothing.